### PR TITLE
Refactored end_state

### PR DESCRIPTION
**Pull request by HowieG**
## 📥 Pull Request

**📘 Description**
end_state being passed as Pydantic Field and then filtered out in serialization. So never gets a default value. 

**🎯 Goal**
Fix end_state in end_session

**🔍 Additional Context**
Makes sense just to remove Pydantic altogether as this is the last place we use it.

### CODE REVIEW

File: agentops/client.py [hunk 1]
```diff
@@ -222,22 +222,24 @@ def start_session(self, tags: Optional[List[str]] = None, config: Optional[Confi
...
+    def end_session(self,
+                    end_state: str,
                     end_state_reason: Optional[str] = None,
                     video: Optional[str] = None):
         """
         End the current session with the AgentOps service.
 
         Args:
-            end_state (str, optional): The final state of the session.
+            end_state (str): The final state of the session.
```
Comment (areibman) [inline agentops/client.py]: Add in docstring the possible options

File: agentops/client.py [hunk 2]
```diff
@@ -288,7 +290,7 @@ def handle_exception(exc_type, exc_value, exc_traceback):
             # Then call the default excepthook to exit the program
             sys.__excepthook__(exc_type, exc_value, exc_traceback)
 
-        atexit.register(lambda: cleanup(end_state="Success", end_state_reason="Process exited normally"))
+        atexit.register(lambda: cleanup(end_state="Indeterminate", end_state_reason="Process exited normally"))
```
Comment (areibman) [inline agentops/client.py]: Maybe change message to something else. "Normally" implies some intention, but the state is indeterminate

**Review by areibman — DISMISSED** [review]
I think we can have a quick discussion about this so everyone is on the same page

**Review by siyangqiu — APPROVED** [review]
I like this

**Pull request merged**